frameworks/Scheme/racket/benchmark_config.json

```diff
@@ -0,0 +1,30 @@
+{
+    "framework": "racket",
+    "tests": [
+        {
+            "default": {
+                "plaintext_url": "/plaintext",
+                "json_url": "/json",
+                "fortune_url": "/fortunes",
+                "db_url": "/db",
+                "query_url": "/queries?queries=",
+                "update_url": "/updates?queries=",
+                "port": 8080,
+                "approach": "Stripped",
+                "classification": "Micro",
+                "database": "Postgres",
+                "framework": "Racket",
+                "language": "Racket",
```

Comment:
Could you change this to `Scheme` just to be consistent? If you'd rather distinguish that as language `Racket`, then I'm ok with a `frameworks/Racket/racket` directory.

Reply:
Maybe we should rename the `Lisp` directory as well to `Common Lisp`!?